### PR TITLE
IN 874 - updating untagged filter

### PR DESCRIPTION
**Pull request by timc-mozilla**
## Summary
* Added `_untagged_` to filter no tags in archive

## References 
* [IN-874](https://getpocket.atlassian.net/jira/software/projects/IN/boards/80?selectedIssue=IN-874)

## Implementation Details
* handle untagged from graphql

## Test Steps
* go to archives
* sort by tag
* select untagged

## PR Checklist:
- [x] Added Unit / UI tests
- [x] Self Review (review, clean up, documentation, run tests)
- [x] Basic Self QA

### CODE REVIEW

File: Tests iOS/MyList/ArchiveFiltersTests.swift [hunk 1]
```diff
@@ -104,4 +104,23 @@ class ArchiveFiltersTests: XCTestCase {
         app.myListView.selectedTagChip(for: "not tagged").buttons.element(boundBy: 0).tap()
         XCTAssertEqual(app.myListView.wait().itemCells.count, 2)
     }
+
```
Comment (timc-mozilla) [inline Tests iOS/MyList/ArchiveFiltersTests.swift:107]: I don't think the test above is testing the name of the test.

Comment (cyndichin) [reply Tests iOS/MyList/ArchiveFiltersTests.swift:107]: Good observation! Some background on it:
The naming was to be consistent with "Favorites" and "All" test names and was not created with the idea of only testing `not tagged`, but any tag value. In general, wanted to test the functionality of the `Tagged` filter. I just decided to choose `not tagged` at that time, but we could have implemented using `tag 0` too. For UI tests, they take longer to run than Unit tests typically so there's a balance in covering all cases and in this case just wanted to test generally that filtering works from UI perspective. However, open to changing the name if that makes more sense. If no differences besides the name, then we should only keep one of the test. Hope that clarifies the reasoning :)

File: PocketKit/Sources/Sync/Operations/ArchiveService.swift [hunk 2]
```diff
@@ -360,7 +360,7 @@ private class FetchArchivePagesOperation: AsyncOperation {
 
             var tagNames: [String] = []
             if let tagName = tagName {
-                tagNames = tagName == "not tagged" ? ["untagged"] : [tagName]
+                tagNames = tagName == "not tagged" ? ["_untagged_"] : [tagName]
```
Comment (cyndichin) [inline PocketKit/Sources/Sync/Operations/ArchiveService.swift:363]: 🔥

File: Tests iOS/MyList/ArchiveFiltersTests.swift (hunk 1 above)
Comment (cyndichin) [reply Tests iOS/MyList/ArchiveFiltersTests.swift:107]: @timc-mozilla Let's keep one of the tests so we're not spending time to run dup tests, else ready for approval!

**Review by cyndichin — CHANGES_REQUESTED** [review]
Looks great! - lets move analytics to another PR as don't want to block this ticket. Just one comment in removing one of the duplicate tests

**Review by cyndichin — APPROVED** [review]
thanks @timc-mozilla :) like that you decided to test both cases

**Pull request merged**